### PR TITLE
FireStore: clearPersistence().

### PR DESCRIPTION
**Pull request by vanniktech**
There are no tests for firestore yet, right?

Sorry that all my commits / PR's are based on others but I'm using a snapshot currently and by basing commits upon commits it's easier for me.

### CODE REVIEW

File: firebase-firestore/src/iosMain/kotlin/dev/gitlive/firebase/firestore/firestore.kt [hunk 1]
```diff
@@ -38,6 +38,9 @@ actual class FirebaseFirestore(val ios: FIRFirestore) {
 
     actual suspend fun <T> runTransaction(func: suspend Transaction.() -> T) =
         awaitResult<Any?> { ios.runTransactionWithBlock({ transaction, error -> runBlocking { Transaction(transaction!!).func() } }, it) } as T
+
+    actual suspend fun clearPersistence() =
+        await { ios.clearPersistenceWithCompletion { } }
```
Comment (nbransby) [inline firebase-firestore/src/iosMain/kotlin/dev/gitlive/firebase/firestore/firestore.kt:43]: need to pass `it` as the completion block to `clearPersistenceWithCompletion`

**Comment by nbransby** [comment]
Yes no tests yet, but you can add the first :) just call the method without any asserts, will be enough for now

**Comment by vanniktech** [comment]
Will do it on Thursday or Friday. Currently, working on something else.

**Comment by nbransby** [comment]
@vanniktech fyi CI for tests added now

**Comment by vanniktech** [comment]
Yep. Haven't been working on that project that uses this library though for quite some time. Will pick it up again once I do, if anyone wants to feel free to continue this PR.

**Pull request merged**